crates/tests-integration/src/container.rs

```diff
@@ -36,8 +40,34 @@ pub(crate) fn test_bootc_install_config() -> Result<()> {
     let config = cmd!(sh, "bootc install print-configuration").read()?;
     let config: serde_json::Value =
         serde_json::from_str(&config).context("Parsing install config")?;
-    // Just verify we parsed the config, if any
-    drop(config);
+    // check that it parses okay, but also ensure kargs is not available here (only via --all)
+    assert!(config.get("kargs").is_none());
+    Ok(())
+}
+
+pub(crate) fn test_bootc_install_config_all() -> Result<()> {
+    #[derive(Deserialize)]
+    struct TestInstallConfig {
+        kargs: Vec<String>,
+    }
+
+    let config_d = std::path::Path::new("/run/bootc/install/");
```

Comment:
This is fine, though I guess probably what we really need here is a workflow that combines container build + test.